pkgs/by-name/fa/falkor/package.nix

```diff
@@ -0,0 +1,60 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchurl,
+  electron,
+  dpkg,
+  makeWrapper,
+  commandLineArgs ? "",
+}:
+
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "falkor";
+  version = "0.0.92";
+
+  src = fetchurl {
+    url = "https://github.com/Team-Falkor/falkor/releases/download/v${finalAttrs.version}/falkor.deb";
+    hash = "sha256-yDpYu2ehrRQuD29jcyTQla2R2IT1zfBDeWDDRnmqc8Y=";
+  };
+
+  unpackPhase = ''
+    runHook preUnpack
+    dpkg -x $src ./
+    runHook postUnpack
+  '';
```

Comment:
```suggestion
```

Just having `dpkg` present in `nativeBuildInputs` will unpack the `src`, so no need to do this.

It can be a bit surprising.